datadog_checks_dev/datadog_checks/dev/tooling/commands/meta/snmp/generate_profile.py

```diff
@@ -9,6 +9,8 @@
 
 import click
 import yaml
+from pysmi.reader.httpclient import HttpReader
```

Comment:
You can import these two within `_get_reader_from_url` and `_get_reader_from_source` to avoid having global dependency on `pysmi` (this is why we import everything `pysmi` related in `_compile_mib_to_json`